docs/content/preview/yugabyte-platform/security/_index.md

```diff
@@ -13,90 +13,48 @@ weight: 660
-      </div>
-      <div class="body">
-        Configure client authentication for your universes.
-      </div>
-    </a>
-  </div>
-
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="authorization-platform/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/secure/authorization.png" aria-hidden="true" />
-        <div class="title">Authorization</div>
-      </div>
-      <div class="body">
-        Manage universe users and roles.
-      </div>
-    </a>
-  </div>
-
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="enable-encryption-at-rest/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/secure/tls-encryption.png" />
-        <div class="title">Enable encryption at rest</div>
-      </div>
-      <div class="body">
-        Use YugabyteDB Anywhere to enable encryption at rest and protect data in storage.
-      </div>
-    </a>
-  </div>
-
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="enable-encryption-in-transit/">
-      <div class="head">
-       <img class="icon" src="/images/section_icons/secure/tls-encryption.png" />
-        <div class="title">Enable encryption in transit (TLS)</div>
-      </div>
-      <div class="body">
-        Use YugabyteDB Anywhere to enable encryption in transit using TLS and protect data between nodes and clients.
-      </div>
-    </a>
-  </div>
-
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="create-kms-config/aws-kms/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/secure/tls-encryption/server-to-server.png" aria-hidden="true" />
-        <div class="title">Create a KMS configuration</div>
-      </div>
-      <div class="body">
-        Configure a key management service with a customer managed key to use for encryption at rest.
-      </div>
-    </a>
-  </div>
-
-</div>
+{{<index/block>}}
+
+  {{<index/item
+    title="Security checklist"
+    body="Address security requirements for YugabyteDB Anywhere and YugabyteDB universes."
+    href="security-checklist-yp/"
+    icon="/images/section_icons/secure/checklist.png">}}
+
+  {{<index/item
+    title="Configure ports"
+    body="Configure YugabyteDB ports for security purposes."
+    href="customize-ports/"
+    icon="/images/section_icons/index/secure.png">}}
+
+  {{<index/item
+    title="Database authentication"
+    body="Configure client authentication for your universes."
+    href="authentication/"
+    icon="/images/section_icons/secure/authentication.png">}}
+
+  {{<index/item
+    title="Database authorization"
+    body="Manage universe users and roles."
+    href="authorization-platform/"
+    icon="/images/section_icons/secure/authorization.png">}}
+
+  {{<index/item
+    title="Enable encryption at rest"
```

Comment:
Change the order. _Encryption in transit_ comes before _Encryption at rest_  in the left nav.